sale_order_line_total_fields/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+Total fields on Sale Orders:
+ -Quantity
```

Comment:
Improve a little bit, for this field, the provided description: note to users that obtained value should be only coherent if every product UoM for the lines is the same